assets/scss/moj-blocks-general.scss

```diff
@@ -1,3 +1,11 @@
+.hale-page {
+  // Deals with the Hero and the highlight/shaded blocks (e.g. CTA, highlighted list)
+  // The shading for these have (need) widths of over 100vw which leads to undesirable horizontal scrolling
+  // No content is outside of the page area, just highlighting.
+  // This prevents this from being an issue by preventing horizontal scroll
+  overflow-x: hidden;
+}
+
```

Comment:
Can you reuse one of the default WP generated classes like `hale-header` or if it's specifically for a page or site, `page-header-default` /  `hale-site-5`?